src/seedu/addressbook/Main.java

```diff
@@ -39,10 +41,20 @@ public static void main(String... launchArgs) {
     public void run(String[] launchArgs) {
         start(launchArgs);
         runCommandLoopUntilExitCommand();
+        printTaggingList();
         exit();
     }
-
     /**
+     * Prints out all tags that were added or removed in this session
+     */
+    private void printTaggingList() {
+		ui.showToUser(MESSAGE_PRINT_TAGGINGLIST);
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.